lib/awful/util.lua

```diff
@@ -512,7 +512,7 @@ function util.table.iterate(t, filter, start)
     return function ()
         while count < length do
             local item = t[index]
-            index = util.cycle(#t, index + 1)
+            index = gmath.cycle(#t, index + 1)
```

Comment:
@blueyed Either way mine or this PR is merged something has to be changed. I'd like to get the tables PR in first (and @Elv13 added an extra commit for the warnings). After that I'm free to start moving everything over to the new gears modules. Small PRs at a time.

I'm not sure why some aren't green anymore, they were all green when I rebased them to be merged in order. Overall I'm okay with rebasing if necessary. I don't know which would be considered easier though, removing that line from the commit or rebasing the PR. :)